packages/office-ui-fabric-react/src/components/ComponentExamples.test.tsx

```diff
@@ -70,6 +70,7 @@ const excludedExampleFiles: string[] = [
   'ExampleHelper.tsx', // Helper file with no actual component
   'GroupedList.Basic.Example.tsx',
   'GroupedList.Custom.Example.tsx',
+  'HoverCard.InstantDismiss.Example.tsx',
```

Comment:
Had to place this new example under excludeExampleFiles list due to some similar errors with some other files in this list. I believe it has to do something with setting __events__ on the instance I could not figure it out.

Reply:
Found the issue and put a comment next to the excluded file.